turtlebot3c-bringup-snap/snap/hooks/post-refresh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+logger -t $SNAP_NAME "Post refresh hook starting."
+
+auto_update=$(snapctl get opencr-auto-update)
+if [ "$auto_update" = "false" ]; then
+  logger -t $SNAP_NAME "OpenCR auto update is disabled. Skipping the update."
+  exit 0
+fi
+
+. "$SNAP/usr/bin/firmware_hash.sh"
+is_hash_the_same
+if [ $? -eq 0 ]; then
```

Comment:
Could this be `if is_hash_the_same; then` ?

Reply:
Tested and it works. I updated the script.